documentation/modules/exploit/unix/webapp/trixbox_ce_endpoint_devicemap_rce.md

```diff
@@ -0,0 +1,90 @@
+## Introduction
+
+This module exploits a post-authentication OS command injection vulnerability found in Trixbox CE <= v2.8.0.4 in the "network" POST parameter of the "/maint/modules/endpointcfg/endpoint_devicemap.php" page. Successful exploitation allows for arbitrary command execution on the underlying operating system as the "asterisk" user.
+
+### Installation And Setup
+
+Trixbox CE v2.8.0.4 Inatallation: 
+
+  1. Download the Trixbox CE v2.8.0.4 ISO (https://netcologne.dl.sourceforge.net/project/asteriskathome/trixbox%20CE/trixbox%202.8/trixbox-2.8.0.4.iso).
+  2. Start Trixbox using the Virtualbox or Vmware player (i.e. the CentOS Linux is starting, the trixbox services are then launched)
+  3. Login at Linux level with the following default credentials (```root```:```trixbox```)
+  4. Configure the IP settings (i.e ```netconfig```)
```

Comment:
This needs to be expanded a bit, still vauge. What exactly needs to be configured in the IP settings? People might not be familiar with `netconfig` or how it works, so I'd like more detail on this if possible.

Reply:
Updated

Reply:
Yep, this is still not what I'm after. Whats the commands, whats the idea.

Like if I say use `ifconfig` to get your PC connected to the mainframe, you're going to ask me what commands and flags I used with `ifconfig` to do that. Same situation here. We have a command name, but you haven't provided me with the exact command you used to set this step up, which is what we are after.

Reply:
Updated